src/BrightScriptDebugSession.ts

```diff
@@ -810,6 +829,10 @@ export class BrightScriptDebugSession extends DebugSession {
         this.sendResponse(response);
     }
 
+    /**
+     * Creates and registers the main events for the RokuAdapter
+     * @param {string} host ip address to connect to
```

Comment:
for typescript, you don't need to add the data type `{string}` in jsdoc comments because the type is already inferred.